### PR TITLE
#8008: Improve visibility of 3d tiles

### PR DESCRIPTION
**Pull request by allyoucanmap**
## Description

This PR includes following visibility fixes:

- fix min and max scale limits visibility
- remove fog and ground atmosphere by default
- allow to apply an offset to the height of the whole tileset

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Bugfix

## Issue

**What is the current behavior?**

#8008

**What is the new behavior?**

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)

 - [x] No

### CODE REVIEW

File: web/client/components/map/cesium/plugins/ThreeDTilesLayer.js [hunk 1]
```diff
@@ -18,6 +18,17 @@ function getStyle({ style }) {
     return Promise.resolve(null);
 }
 
+function updateModelMatrix(tileSet, { heightOffset }) {
```
Comment (MV88) [inline web/client/components/map/cesium/plugins/ThreeDTilesLayer.js:21]: you miss a third parameter here?

File: web/client/translations/data.vi-VN.json [hunk 2]
```diff
@@ -697,6 +697,7 @@
             "textFormatTitle": "VĂN BẢN",
             "title": "Tiêu đề",
             "titleTranslations": "Bản dịch tiêu đề",
+            "heightOffset": "Height offset (m)",
```
Comment (MV88) [inline web/client/translations/data.vi-VN.json:700]: no need to add locales messages, even if not translated, to not supported locales :)

**Review by MV88 — CHANGES_REQUESTED** [review]
just a minor observation

**Review by MV88 — APPROVED** [review]

**Comment by MV88** [comment]
@ElenaGallo please test in DEV

**Comment by ElenaGallo** [comment]
@allyoucanmap @tdipisa when the offset number is deleted the gray error page appears.
m

**Pull request merged**